build/containers/Dockerfile.webhook-configuration

```diff
@@ -0,0 +1,24 @@
+ARG PLATFORM=amd64
+ARG CROSS_BUILD_TARGET=x86_64-unknown-linux-gnu
+
+FROM ${PLATFORM}/debian:buster-slim
+
+ARG CROSS_BUILD_TARGET
+
+RUN echo "Creating container based on ${PLATFORM}/debian:buster-slim"
```

Comment:
Would suggest combining all RUN directives to one to reduce the number of layers in the final container image. For example, you could:

RUN echo foo && echo bar && apt-get update ...

Reply:
Yes. I followed the model using the other Dockerfiles. We should probably have an issue to update them all.